src/DotVVM.Framework/Controls/HtmlGenericControl.cs

```diff
@@ -194,6 +200,31 @@ private void AddCssClassesToRender(IHtmlWriter writer)
             if (cssClassBindingGroup != null) writer.AddKnockoutDataBind("css", cssClassBindingGroup);
         }
 
+        private void AddStylePropertiesToRender(IHtmlWriter writer)
+        {
+            KnockoutBindingGroup stylePropertiesBindingGroup = null;
+            foreach (var styleProperty in StyleProperties.Properties)
+            {
+                if (stylePropertiesBindingGroup == null) stylePropertiesBindingGroup = new KnockoutBindingGroup();
+                stylePropertiesBindingGroup.Add(styleProperty.GroupMemberName, this, styleProperty);
```

Comment:
This will add hard-coded values to binding group. I think it's not necessary because hard-coded values are added to `style` attribute and can't be changed.